lib/checks/shared/filename-is-valid-accessible-name.js

```diff
@@ -0,0 +1,29 @@
+const { text } = axe.commons;
+const possibleImageExtensions = [`.jpg`, `.jpeg`, `.png`, `.gif`, `.tif`];
```

Comment:
Should include WebP images as well

Reply:
This should be an option to the check with a default value in the JSON file